common/snapshots.py

```diff
@@ -3104,6 +3104,25 @@ def lastSnapshot(cfg):
         return sids[0]
 
 
+def hasMissing(included):
+    """
+    Check if there are missing files or folders in a snapshot.
+
+    Args:
+        included (list):    list of tuples (item, info)
+
+    Returns:
+        tuple:              (bool, str) where bool is ``True`` if there are
+                            missing files or folders and str is a message
+                            describing the missing files or folders
+    """
+    notFound = []
+    for path, info in included:
+        if not os.path.exists(path):
+            notFound.append(path)
+    return bool(notFound), notFound
```

Comment:
Mhm... I would prefer just to return the resulting list `notFound`. The caller then can just to `if missing:` on this list object. An empty list will "return" `False`.